modules/monitoring-configuring-external-alertmanagers.adoc

```diff
@@ -6,24 +6,34 @@
 [id="monitoring-configuring-external-alertmanagers_{context}"]
 = Configuring external Alertmanager instances
 
-The {product-title} monitoring stack includes a local Alertmanager instance that routes alerts from Prometheus.
-You can add external Alertmanager instances by configuring the `cluster-monitoring-config` config map in either the `openshift-monitoring` project or the `user-workload-monitoring-config` project.
+The {product-title} monitoring stack includes a local Alertmanager instance that routes alerts from Prometheus. You can add external Alertmanager instances by configuring the `cluster-monitoring-config` config map in
```

Comment:
from https://60762--docspreview.netlify.app/openshift-rosa/latest/monitoring/configuring-the-monitoring-stack.html#monitoring-configuring-external-alertmanagers_configuring-the-monitoring-stack
`You can add external Alertmanager instances by configuring the cluster-monitoring-config config map in the user-workload-monitoring-config project.`
should be
`You can add external Alertmanager instances by configuring the user-workload-monitoring-config config map in the openshift-user-workload-monitoring project.`

Reply:
Done.